### ISSUE TITLE
Add support for "Remote Development" feature of vscode.

### ISSUE THREAD

**Issue by Dontar** [mhutchie/vscode-git-graph#83]
With the upcoming "Remote Development" feature in VSCode, it would be nice to have support for it. As far as I could investigate "Remote Development" introduces the concept of `local` and `remote` extensions. In order to be compatible this extension needs to implement the `remote` paradigm. With that said the compatibility is already there except for the `WebviewPanel` used in `gitGraphView.ts`.
In short, the way I managed to make it work was to inline the files in `media` folder directly in the html of the WebviewPanel. I'm not sure that this is particularly elegant solution considering that I haven't researched the extension in its entirety.

**Comment by mhutchie** [comment]
Thanks for this suggestion & initial implementation! It would definitely be nice to support the upcoming "Remote Development" functionality. After having a look at the code you raised a PR for, it will definitely result in a performance decrease due to the three blocking synchronous file system operations on the initial load. A possible solution would be to convert each file system operation to be async, wrap them each in a promise, and then await for Promise.all of them, then you can just pass in the file contents inline to the Webview content as you have done.

I'd like to focus on the six outstanding new features that I have committed to releasing by the end of this week (24/05/2019), as "Remote Development" isn't available in the standard Visual Studio Code (for at least the next two weeks). If I have spare time before the 1.6.0 release, or as soon as 1.6.0 is released, I'll look into ways alternative ways of supporting "Remote Development" without introducing a noticeable performance overhead (I think the Promise.all solution should be fine if you're interested in trying it out).

I intend to release 1.7.0 by the end of the following week (31/05/2019), which would make "Remote Development" definitely available before it is released in the standard Visual Studio Code if it isn't able to make 1.6.0 (which I'll still aim for).

**Comment by mhutchie** [comment]
This will be available in 1.6.0, which will be released in the next day. Thank you for helping out with the development of this.

**Comment by ToonoW** [comment]
When I used it today, it recovered itself and it can be used again.

**Comment by mhutchie** [comment]
Microsoft fixed the issue preventing Git Graph from working. I'll roll back these changes before releasing 1.6.0 today.
